pkg/operator/controller/status/controller.go

```diff
@@ -140,7 +140,14 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 
 	co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorAvailableCondition(allIngressesAvailable))
 	co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorProgressingCondition(allIngressesAvailable, oldStatus.Versions, co.Status.Versions, r.OperatorReleaseVersion, r.IngressControllerImage))
-	co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorDegradedCondition(state.IngressControllers))
+	co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorIngressDegradedCondition(state.IngressControllers))
+
+	authStateObj := &operatorv1.Authentication{ObjectMeta: metav1.ObjectMeta{Name: "cluster"}}
+	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: "authentication"}, authStateObj); err != nil {
+		log.Error(err, "unable to get authentication cluster operator")
+	} else {
+		co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorAuthenticationDegradedCondition(authStateObj.Status))
```

Comment:
Doesn't this conflict with the conditions created by computeOperatorDegradedCondition resulting in a last-in-wins scenario?

I guess I would have expected computeOperatorDegradedCondition to be changed to incorporate new criterion (currently just ingress controllers, now ingress controllers and the authentication operator status)

Reply:
i would think this is going to be merged to 1 degraded=true with two conditions? at least that is how it works in control plane operator //cc @deads2k

Reply:
@ironcladlou also the computeOperatorDegradedCondition only return 1 condition, right? it might be we have multiple in this case

Reply:
The current merge logic will not aggregate multiple conditions of the same type. What you're saying might be better, but that's not what it does today